tools/azsdk-cli/Azure.Sdk.Tools.Cli/Configuration/RequirementsV1.json

```diff
@@ -0,0 +1,121 @@
+{
+    "categories": {
+        "core": [
+            {   
+                "requirement": "Node.js >= 22.16.0",
+                "check": ["node", "--version"],
+                "instructions": ["Download Node.js"]
+            },
+            {
+                "requirement": "npm",
+                "check": ["npm", "--version"],
+                "instructions": ["Download Node.js"]
+            },
+            {
+                "requirement": "tsp-client >= 0.24.0",
+                "check": ["tsp-client", "--version"],
+                "instructions": ["npm install -g @azure-tools/typespec-client-generator-cli@latest @typespec/compiler@latest"]
+            },
+            {
+                "requirement": "tsp >= 1.0.0",
+                "check": ["tsp", "--version"],
+                "instructions": ["npm install -g @azure-tools/typespec-client-generator-cli@latest @typespec/compiler@latest"]
+            },
+            {
+                "requirement": "Powershell >= 7",
+                "check": ["pwsh", "--version"],
+                "instructions": ["Download and install PowerShell"]
+            },
+            {
+                "requirement": "github CLI >= 2.30.0",
+                "check": ["gh", "--version"],
+                "instructions": ["Download and install GitHub CLI"]
+            }
+        ],
+        "java": [
+            {
+                "requirement": "java >= 17",
```

Comment:
I believe java 11 would be enough. Add @weidongxu-microsoft for confirmation

Reply:
Higher version would definitely work. We can have this as 17, till we have feedback from users.